src/platform_impl/linux/event_loop.rs

```diff
@@ -600,6 +628,21 @@ impl<T: 'static> EventLoop<T> {
                 }) {
                   log::warn!("Failed to send keyboard event to event channel: {}", e);
                 }
+
+                if let Ok(shortcuts) = shortcuts_.try_borrow() {
+                  for accelerator in (*shortcuts).values() {
+                    if accelerator.mods.contains(mods) && accelerator.key == key {
+                      if let Err(e) =
+                        tx_clone.send(Event::GlobalShortcutEvent(accelerator.clone().id()))
+                      {
+                        log::warn!(
+                          "Failed to send global shortcut event to event channel: {}",
+                          e
+                        );
+                      }
+                    }
+                  }
+                }
```

Comment:
GlobalShortcuts are supposed to be "Global" and not tied to a specific window, this code is ran inside a window specific event though. It should be possible to have a tray-only app and still possible to have GlobalShortcut.

Reply:
Oh no. There's no such signal for GtkApplication. I think I'll have to find other way or maybe back for GAction. :(

Reply:
if you want, you can have a hidden window for the non-window events (DeviceEvents), kinda like how it is done on Windows. In `src/platform_impl/windows/event_loop.rs` there is a `subclass_event_target_window()` function which has a different event loop logic than the one used for windows created by the user.

Reply:
I tried GAction instead. It's the original plan but the API only accepts strings. I wire through `key_to_raw_key` to convert them. So there might be some keys missing.